plugins/org.eclipse.papyrus.uml.interaction.model/src-gen/org/eclipse/papyrus/uml/interaction/internal/model/impl/MExecutionImpl.java

```diff
@@ -147,6 +155,93 @@ public MLifeline getOwner() {
 		return super.getDiagramView().map(Shape.class::cast);
 	}
 
+	/**
+	 * <!-- begin-user-doc --> <!-- end-user-doc -->
+	 * 
+	 * @generated NOT
+	 */
+	@Override
+	public CreationCommand<ExecutionSpecification> insertNestedExecutionAfter(MElement<?> before, int offset,
+			int height, Element specification) {
+		return new InsertNestedExecutionCommand(this, before, offset, height, specification);
+	}
+
+	/**
+	 * <!-- begin-user-doc --> <!-- end-user-doc -->
+	 * 
+	 * @generated NOT
+	 */
+	@Override
+	public CreationCommand<ExecutionSpecification> insertNestedExecutionAfter(MElement<?> before, int offset,
+			int height, EClass metaclass) {
+		return new InsertNestedExecutionCommand(this, before, offset, height, metaclass);
+	}
+
+	/**
+	 * <!-- begin-user-doc --> <!-- end-user-doc -->
+	 * 
+	 * @generated NOT
+	 */
+	@Override
+	public Optional<MElement<? extends Element>> elementAt(int offset) {
+		int top = layoutHelper().getTop(getShape(getElement()));
+		int absoluteOffset = top + offset;
+
+		Predicate<MElement<?>> isAtOrAbove = e -> e.getBottom().orElse(Integer.MAX_VALUE) <= absoluteOffset;
+
+		Optional<? extends MElement<? extends Element>> result = getVertex().flatMap(vtx -> //
+		vtx.successors().sequential().filter(covers(getOwner().getElement())) //
+				.map(Vertex::getInteractionElement) //
+				.map(getInteraction()::getElement) //
+				.filter(Optional::isPresent).map(Optional::get) //
+				.filter(isAtOrAbove)//
+				.reduce((a, b) -> b)// last element
+		);
+
+		return Optional.ofNullable(result.orElse(null));
+	}
+
+	/**
+	 * <!-- begin-user-doc --> <!-- end-user-doc -->
+	 * 
+	 * @generated NOT
+	 */
+	@SuppressWarnings("boxing")
+	@Override
+	public List<MExecution> getNestedExecutions() {
+		List<MExecution> nestedExecutions = new ArrayList<>();
+		// browse all executions on lifeline after the start of this execution and before the end of this
+		// execution
+
+		List<MOccurrence<?>> occurences = getOwner().getOccurrenceSpecifications();
```

Comment:
Typo:  should be 'occur**r**ences'.  Similarly for variables 'executionOccurences', 'occurence' below.

Reply:
fixed with commit 348a8d6